bin/sender.py

```diff
@@ -62,40 +62,102 @@ def main():
         sys.exit(1)
     
     log = logging.getLogger('ssmsend')
-    
+
+    # Set defaults for MQ-BROKER only variables
+    use_ssl = None
+    # Set defaults for AMS only variables
+    token = ""
+    project = None
```

Comment:
Already ha default in SSM class

Reply:
Which is irrelevant as it still needs to be set to pass to the `Ssm2` constructor. However, most of these settings are fetched from config with exception handling if they're not present.